### PR TITLE
Add share component

### PR DESCRIPTION
**Pull request by jpppovoa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /node_modules
 dist
 .idea
+Trackovid19-web/src/environments/environment.ts
```
Comment (bmm) [inline .gitignore]: Retira do ignore pf

**Review by bmm — CHANGES_REQUESTED** [review]

**Review by bmm — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (jpppovoa) [reply .gitignore]: done tks

**Comment by fabiofcferreira** [comment]
I refactored the desktop view (social share buttons, paddings, units), adapted the modal to mobile screens and fixed a bug with the modal open status.
It's ready to be merged IMHO.

**Comment by fabiofcferreira** [comment]
Added redirect from change status step 3 to dashboard view with the modal being automatically displayed (using query parameter `share=true`)

**Pull request merged**